contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -92,6 +93,48 @@ contract LidoOracle is ILidoOracle, AragonApp {
     bytes32 internal constant LAST_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.lastCompletedEpochId");
     bytes32 internal constant PREV_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.prevCompletedEpochId");
 
+    /*
+    If we use APR as a basic reference for increase, and expected range is between 1 and 10% APR.
+    So the granularity step for yearly APR is 1%
+    daily_granularity = 0.01 / 365 = 2.74e-05
+    Currently used basisPoints (1e-4) is unable to express such microscopic ratio.
+    So we should take another popular metric to express this - PPM or 1e-6
+    daily_granularity_in_ppm = 2.74e-05 / 1e-6 = 27 (pretty good)
+    with 100% buffer for future spikes it will be 50.
+    May be changed by Gov.
+    */
+    bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION =
+        keccak256("lido.LidoOracle.allowedBeaconBalanceAnnualRelativeIncrease");
+    uint256 public constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE = 50;  // PPM
+
+    /*
+    When slashing happens, the balance may decrease at a much faster pace.
+    On the start the system will have about 100 validators and we can guess one of them can be slashed for any reason.
+    1/100 = 10000 PPM.
+    May be changed by Gov.
+    */
+    bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_DECREASE_POSITION =
+        keccak256("lido.LidoOracle.allowedBeaconBalanceAnnualDecrease");
+    uint256 public constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_DECREASE = 50000;  // 5% ~ 50000 PPM
+
+    function getAllowedBeaconBalanceAnnualRelativeIncrease() public pure returns(uint256) {
+        return ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION.getStorageUint256(value);
+    }
+
+    function getAllowedBeaconBalanceAnnualRelativeDecrease() public pure returns(uint256) {
+        return ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_DECREASE_POSITION.getStorageUint256(value);
+    }
+
+    function setAllowedBeaconBalanceAnnualRelativeIncrease(uint256 value) external auth(SET_REPORT_BOUNDARIES) {
+        ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION.setStorageUint256(value);
+        emit SetAllowedBeaconBalanceAnnualRelativeIncrease(value);
```

Comment:
I think it is better to use imperative style: `AllowedBeaconBalanceAnnualRelativeIncreaseSet`.

Reply:
thnx, done